ConcernsCaseWork/ConcernsCaseWork/Startup.cs

```diff
@@ -103,6 +99,9 @@ public void ConfigureServices(IServiceCollection services)
 				options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
 			});
 
+			// Add Health Checks
+			services.AddHealthChecks();

```

Comment:
Why do we have two instances of this? Another one is defined above in the StgartupExtensions class